portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,32 @@
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+body{
+  margin-left: auto;
+  margin-right: auto;
+  width: fit-content;
+}
+
 #greeting-container {
   margin-top: 20px;
 }
+
+#selfie{
+    width: 200px;
+}
+
+#header{
+  display: flex;
+}
+
+#header-right{
+  padding-left: 10px;
+}
+
+#introduction{
+  margin: 0px;
```

Comment:
nit: you can omit the unit if it's 0.
https://google.github.io/styleguide/htmlcssguide.html#0_and_Units